src/components/icon/icon.tsx

```diff
@@ -237,7 +237,7 @@ const typeToPathMap = {
   logoCouchbase: 'logo_couchbase',
   logoDocker: 'logo_docker',
   logoDropwizard: 'logo_dropwizard',
-  logoElastic: 'logo_elastic',
```

Comment:
Should we keep this for the sake of backwards compatibility? Do we know if anyone else is using this?

Reply:
I don't think this change is in scope of this PR - there are several edits to the logos file that I am tracking as part of the meta issue for compliance.

Reply:
As per last task (Change logoElastic to logoOUI) from #111, I changed all `logoElastic` references to `logoOUI`.  Does it mean that in this PR we should just update `logoElastic` to `logoOUI` and keep name of logo which is `logo_elastic` as it is in this PR?